snmp_types.yaml

```diff
@@ -0,0 +1,87 @@
+tosca_definitions_version: cloudify_dsl_1_0
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.2m4/types.yaml
```

Comment:
`types.yaml` should not be imported in the `snmp-types.yaml` as it should be version agnostic. The main blueprint file should be the one importing it